openshift/e2e-common.sh

```diff
@@ -64,69 +64,23 @@ function install_strimzi(){
 
 function install_serverless(){
   header "Installing Serverless Operator"
-  git clone --branch release-1.6 https://github.com/openshift-knative/serverless-operator.git /tmp/serverless-operator
+  local operator_dir=/tmp/serverless-operator
+  local failed=0
+  git clone --branch release-1.8 https://github.com/openshift-knative/serverless-operator.git $operator_dir
+  #cp openshift/olm/serverless-operator.clusterserviceversion.yaml $operator_dir/olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml
+  cp openshift/serverless.bash $operator_dir/hack/lib/serverless.bash
   # unset OPENSHIFT_BUILD_NAMESPACE as its used in serverless-operator's CI environment as a switch
   # to use CI built images, we want pre-built images of k-s-o and k-o-i
   unset OPENSHIFT_BUILD_NAMESPACE
-  /tmp/serverless-operator/hack/install.sh || return 1
-  header "Serverless Operator installed successfully"
-}
-
-function create_knative_namespace(){
-  local COMPONENT="knative-$1"
-
-  cat <<-EOF | oc apply -f -
-	apiVersion: v1
-	kind: Namespace
-	metadata:
-	  name: ${COMPONENT}
-	EOF
-}
-
-function deploy_knative_operator(){
```

Comment:
This function is being deleted. But I don't see the place where it's called in this PR. So it might still be called? Not sure

Reply:
not being called, it was mostly dead code being cleaned up.

Reply:
yep, we no longer need it